src/components/Board.tsx

```diff
@@ -0,0 +1,90 @@
+import { useEffect, useState, useCallback } from "react";
+import { Square } from "./Square";
+import "../css/board.css";
+
+interface SquareInterface {
+    pieceColor: string,
+    color: string,
+    piece: string
+}
+
+export function Board() {
+    let [squares, setSquares] = useState<Array<Array<SquareInterface>>>();
+    let [firstRowOrder] = useState([
+        "rook",
+        "knight",
+        "bishop",
+        "queen",
+        "king",
+        "bishop",
+        "knight",
+        "rook",
+    ]);
+
+    const getPieceColor = (row: number) => {

```

Comment:
This should be defined outside of the component. As it is, it would get re-defined on every render of the Board.